### PR TITLE
Add support for `spending_limits` with Issuing Card and Cardholder

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @brandur-stripe 
cc @stripe/api-libraries @zacht-stripe

### CODE REVIEW

**Comment by remi-stripe** [comment]
@brandur-stripe Thanks for the review! For some reason I forgot to post my review/comments after I was done yesterday. Any chance you could have a quick look?

File: issuing_card.go [hunk 1]
```diff
@@ -45,12 +45,37 @@ const (
...
+// IssuingAuthorizationControlsSpendingLimitsParams is the set of parameters that can be used for
+// the spending limits associated with a given issuing card or cardholder.
+type IssuingAuthorizationControlsSpendingLimitsParams struct {
+	Amount     *int64    `form:"amount"`
+	Categories []*string `form:"categories"`
+	Interval   *string   `form:"interval"`
+}
+
 // AuthorizationControlsParams is the set of parameters that can be used for the shipping parameter.
 type AuthorizationControlsParams struct {
```
[the remark this answers is not in the document]
Comment (brandur-stripe) [reply issuing_card.go]: Yeah I noticed that too. Yeah, I think it's fine to just wait for the next major.

File: issuing_card.go [hunk 2]
```diff
@@ -98,13 +123,25 @@ type IssuingCardDetails struct {
...
+// IssuingAuthorizationControlsSpendingLimits is the resource representing spending limits
+// associated with a card or cardholder.
+type IssuingAuthorizationControlsSpendingLimits struct {
+	Amount     int64                        `json:"amount"`
+	Categories []string                     `json:"categories"`
+	Interval   IssuingSpendingLimitInterval `json:"interval"`
+}
+
 // IssuingCardAuthorizationControls is the resource representing authorization controls on an issuing card.
+// TODO: Rename to IssuingAuthorizationControls in the next major
```
[the remark this answers is not in the document]
Comment (brandur-stripe) [reply issuing_card.go]: +1.

**Comment by brandur-stripe** [comment]
Thanks for the additional flavor Remi — I'm in agreement on waiting for the next major and just having the TODO for now. LGTM.

**Pull request merged**